manifest_template.yml

```diff
@@ -60,24 +66,21 @@ apigee:
         attributes:
           - name: access
             value: public
-            # For a parameterised example of rate limiting per environment,
-            # see https://github.com/NHSDigital/personal-demographics-service-api/blob/master/manifest_template.yml
           - name: ratelimiting
             value:
               {{ NAME }}:
-                quota:
-                  enabled: true
-                  limit: {{ ENV.quota | default('300') }}
-                  interval: 1
-                  timeunit: minute
-                spikeArrest:
-                  enabled: true
-                  ratelimit: {{ ENV.ratelimit | default('600pm') }} # 10 requests per second default
-              app:
                 quota:
                   enabled: false
                 spikeArrest:
                   enabled: false
+          - name: requestLimit_referral
+            value: {{ ENV.request_limit | default('2880000') }}
+          - name: requestLimit_general
+            value: {{ ENV.request_limit | default('2880000') }}
+          - name: burstLimit_referral
+            value: {{ ENV.burst_limit | default('6300pm') }}
+          - name: burstLimit_general
+            value: {{ ENV.burst_limit | default('6300pm') }}
```

Comment:
From these values looks like there won't be any difference between _referral (ServiceRequest in path) and _general (ServiceRequest not in path)?

Reply:
Currently they're both set to the same value yup, I've left them separate here incase we wanted to apply separate limits but they could be collapsed down to two product variables if we wanted.